### PR TITLE
Use specific keyframes names for sliding panel

### PR DESCRIPTION
**Pull request by RumataDeEstor**
### What does this PR do:

Change keyframes to be specific for ui-kit. Reason: the same keyframes names in RWD which are overriding ui-kit ones and breaking them.

### Where should the reviewer start:

Diff.

### CODE REVIEW

**Comment by rbardini** [comment]
@RumataDeEstor we have a couple other keyframe declarations in UI Kit. Would you mind updating them as well? It would be nice to have them all namespaced to avoid similar problems in the future 🙂

File: packages/travix-ui-kit/components/slidingPanel/slidingPanel.css [hunk 1]
```diff
@@ -34,24 +34,24 @@
...
 }
 
 .ui-sliding-panel_right.ui-sliding-panel_active  {
-  animation: sidePanelOpen 195ms ease-in-out 0s 1 both;
+  animation: uiKitSidePanelOpen 195ms ease-in-out 0s 1 both;
 }
 
 .ui-sliding-panel_left {
-  animation: sidePanelLeftClose 195ms ease-in-out 0s 1 both;
+  animation: uiKitsidePanelLeftClose 195ms ease-in-out 0s 1 both;
```
Comment (leandrooriente) [inline packages/travix-ui-kit/components/slidingPanel/slidingPanel.css]: Missing uppercase in SidePanel

**Comment by RumataDeEstor** [comment]
@rbardini Interesting. For one of them 'bouncedelay' I cannot see the usage in animation. Is it some hack or should I delete it? :D

File: packages/travix-ui-kit/components/slidingPanel/slidingPanel.css [hunk 2]
```diff
@@ -87,7 +87,7 @@
   }
 }
 
-@keyframes sidePanelLeftClose {
+@keyframes uiKitsidePanelLeftClose {
```
Comment (leandrooriente) [inline packages/travix-ui-kit/components/slidingPanel/slidingPanel.css]: The same here

**Comment by rbardini** [comment]
@RumataDeEstor you can delete it 👍

**Comment by RumataDeEstor** [comment]
@leandrooriente @rbardini updated

**Pull request merged**